### ISSUE TITLE
083 additional dewey decimal classification number

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#95]
https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/0XX.csv

**Comment by CECSpecialistI** [comment]
@szapoun

**Comment by CECSpecialistI** [comment]
Review complete. See coding for 080, might be helpful in writing transformation code.

**Comment by Sarahanc** [comment]
@szapoun @CECSpecialistI The transformation notes for 083 include this information about skos:inScheme "tScheme(schemeCode) for 082 ind1=0 = http://id.loc.gov/vocabulary/classSchemes/ + “ddc” + “Full” + $2" is this also applicable to 083?

**Comment by szapoun** [comment]
I think yes, we could do that. Thank you SaraΣτάλθηκε από το iPhone μου23 Ιουν 2025, 12:49 πμ, ο χρήστης «Sarahanc ***@***.***>» έγραψε:﻿Sarahanc left a comment (uwlib-cams/MARC2RDA#95)
@szapoun @CECSpecialistI The transformation notes for 083 include this information about skos:inScheme "tScheme(schemeCode) for 082 ind1=0 = http://id.loc.gov/vocabulary/classSchemes/ + “ddc” + “Full” + $2" is this also applicable to 083?

**Comment by GordonDunsire** [comment]
@Sarahanc: You are correct that the DDC abridged editions can only be mapped to the general DDC scheme. We must assume that the LC IRIs refer to full editions; as far as I know, there is no relationship between abridged editions and full editions.

[The mapping notes originate with my document on Transforming subject data, and I can no longer recall why the boilerplate 'full' and 'abridged' was added :-(
